src/main/scala/BintrayPlugin.scala

```diff
@@ -0,0 +1,194 @@
+package bintray
+
+import bintry.Attr
+import sbt.{ AutoPlugin, Credentials, Global, Path, Resolver, Setting, Task, Tags, ThisBuild }
+import sbt.Def.{ Initialize, setting, task, taskDyn }
+import sbt.Keys._
+import sbt.Path.richFile
+
+object BintrayPlugin extends AutoPlugin {
+  import BintrayKeys._
+  import InternalBintrayKeys._
+
+  override def requires = sbt.plugins.JvmPlugin
+  override def trigger = allRequirements
+  
+  override def globalSettings: Seq[Setting[_]] = globalPublishSettings
+  override def buildSettings: Seq[Setting[_]] = buildPublishSettings
+  override def projectSettings: Seq[Setting[_]] = bintraySettings
+
+  object autoImport extends BintrayKeys {
+  }
+
+  lazy val Git = Tags.Tag("git")
```

Comment:
[Tags](http://www.scala-sbt.org/0.13/docs/Parallel-Execution.html#Tagging+Tasks) are concurrency control similar to [semaphors](http://en.wikipedia.org/wiki/Semaphore_%28programming%29).

On multi-project build with dozens of subprojects, `resolveVcsUrl` gets bombarded with multiple calls to `git`:

```scala
sbt.Process("git" :: "remote" :: "-v" :: Nil).!!
```

I'm turning it into a task with the custom tag called `Git`, and telling the task engine that only one task with the tag is allowed to execute concurrently. This should work like putting locks or critical section around `resolveVcsUrl`, but done sbt way.